stable_baselines3/common/base_class.py

```diff
@@ -167,10 +166,6 @@ def _wrap_env(self, env: GymEnv) -> VecEnv:
                 print("Wrapping the env in a DummyVecEnv.")
             env = DummyVecEnv([lambda: env])
 
-        if is_image_space(env.observation_space) and not isinstance(env, VecTransposeImage):
```

Comment:
This check should be kept but with an exception or a warning if last dimension seems to be channels (e.g. `assert shape[0] < shape[2]`. The whole "channels-first" convention will throw some people off, and in worst cases nothing else will warn about this, apart from CNN layers going to negative dimension shapes :)